engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala

```diff
@@ -1,21 +1,41 @@
 package pl.touk.nussknacker.k8s.manager
 
 import akka.actor.ActorSystem
-import com.typesafe.config.Config
+import com.typesafe.config.{Config, ConfigFactory, ConfigRenderOptions}
 import com.typesafe.scalalogging.LazyLogging
 import pl.touk.nussknacker.engine.api.deployment._
+import pl.touk.nussknacker.engine.api.deployment.simple.SimpleStateStatus
 import pl.touk.nussknacker.engine.api.process.ProcessName
 import pl.touk.nussknacker.engine.api.queryablestate.QueryableClient
 import pl.touk.nussknacker.engine.api.{ProcessVersion, StreamMetaData}
 import pl.touk.nussknacker.engine.lite.kafka.KafkaTransactionalScenarioInterpreter
 import pl.touk.nussknacker.engine.marshall.ScenarioParser
 import pl.touk.nussknacker.engine.version.BuildInfo
 import pl.touk.nussknacker.engine.{DeploymentManagerProvider, ModelData, TypeSpecificInitialData}
-import skuber.{Container, Pod, PodList, k8sInit}
+import pl.touk.nussknacker.k8s.manager.K8sDeploymentManager._
+import skuber.LabelSelector.IsEqualRequirement
+import skuber.LabelSelector.dsl._
+import skuber.apps.v1.Deployment
+import skuber.json.format._
+import skuber.{ConfigMap, Container, EnvVar, LabelSelector, ListResource, ObjectMeta, Pod, Volume, k8sInit}
 import sttp.client.{NothingT, SttpBackend}
 
+import java.util.Collections
 import scala.concurrent.{ExecutionContext, Future}
-
+import scala.language.reflectiveCalls
+
+/*
+  Each scenario is deployed as Deployment+ConfigMap
+  Id of both is created with scenario id + sanitized name (scenario id guarantees id is unique)
+  Labels contain scenario name, scenario id and version
+  ConfigMap contains: model config, overrides for execution and scenario
+  TODO:
+   - better implementations of cancel, status, redeployment
+   - maybe ConfigMap should have version in its id? Can we guarantee correct one is read when redeploying?
+   - more data in annotations? Are they needed?
+   - replica count configuration
+   - label for name is used to cancel/findStatus. We should use ProcessId to avoid uniqueness problems after sanitization
```

Comment:
The problem is described correctly, but have no clue if solution will be good. Can be other options like storing mapping of name -> sanitized name. Maybe we shouldn't fix to solution in TODO on this step?

Reply:
ok, changed